[elinks] error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory

```
$ elinks
/usr/local/bin/elinks: error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory
```
I tried reinstalling from source and received the error below:
```
socket.c:142:37: warning: pointer targets in assignment from ‘char *’ to ‘unsigned char *’ differ in signedness [-Wpointer-sign]
  142 |                         client_cert = getenv("X509_CLIENT_CERT");
      |                                     ^
socket.c:148:61: error: invalid use of incomplete typedef ‘SSL’ {aka ‘struct ssl_st’}
  148 |                         SSL_CTX *ctx = ((SSL *) socket->ssl)->ctx;
      |                                                             ^~
socket.c:150:65: warning: pointer targets in passing argument 2 of ‘SSL_CTX_use_certificate_chain_file’ differ in signedness [-Wpointer-sign]
  150 |                         SSL_CTX_use_certificate_chain_file(ctx, client_cert);
      |                                                                 ^~~~~~~~~~~
      |                                                                 |
      |                                                                 unsigned char *
In file included from socket.c:8:
/usr/local/include/openssl/ssl.h:1610:73: note: expected ‘const char *’ but argument is of type ‘unsigned char *’
 1610 | __owur int SSL_CTX_use_certificate_chain_file(SSL_CTX *ctx, const char *file);
      |                                                             ~~~~~~~~~~~~^~~~
socket.c:151:58: warning: pointer targets in passing argument 2 of ‘SSL_CTX_use_PrivateKey_file’ differ in signedness [-Wpointer-sign]
  151 |                         SSL_CTX_use_PrivateKey_file(ctx, client_cert,
      |                                                          ^~~~~~~~~~~
      |                                                          |
      |                                                          unsigned char *
/usr/local/include/openssl/ssl.h:1605:66: note: expected ‘const char *’ but argument is of type ‘unsigned char *’
 1605 | __owur int SSL_CTX_use_PrivateKey_file(SSL_CTX *ctx, const char *file,
      |                                                      ~~~~~~~~~~~~^~~~
make[3]: *** [../../.././Makefile.lib:115: socket.o] Error 1
make[3]: Leaving directory '/usr/local/tmp/crew/elinks.20220908154234.dir/elinks-0.11.7/src/network/ssl'
make[2]: *** [../.././Makefile.lib:217: all-recursive] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/elinks.20220908154234.dir/elinks-0.11.7/src/network'
make[1]: *** [.././Makefile.lib:217: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/elinks.20220908154234.dir/elinks-0.11.7/src'
make: *** [Makefile.lib:217: all-recursive] Error 1
/usr/local/lib/crew/lib/package.rb:266:in `system': Command failed with exit 2: bash
elinks failed to install: `CFLAGS="-O2 -pipe -ffat-lto-objects -fPIC -fuse-ld=mold  -flto" CXXFLAGS="-O2 -pipe -ffat-lto-objects -fPIC -fuse-ld=mold  -flto" FCFLAGS="-O2 -pipe -ffat-lto-objects -fPIC -fuse-ld=mold  -flto" FFLAGS="-O2 -pipe -ffat-lto-objects -fPIC -fuse-ld=mold  -flto" LDFLAGS="-flto " bash -c make -j8` exited with 2
```
This seems to be a general problem for all packages that depend on openssl.  I had the same issue with links but, fortunately, I was able to update.  See #7387.

--- a/packages/elinks.rb
+++ b/packages/elinks.rb
@@ -3,35 +3,38 @@ require 'package'
 class Elinks < Package
   description 'Full-Featured Text WWW Browser'
   homepage 'http://elinks.or.cz/'
-  version '0.11.7'
+  version '0.13'
   license 'GPL-2'
   compatibility 'all'
-  source_url 'http://elinks.or.cz/download/elinks-0.11.7.tar.bz2'
-  source_sha256 '456db6f704c591b1298b0cd80105f459ff8a1fc07a0ec1156a36c4da6f898979'
+  source_url 'http://elinks.or.cz/download/elinks-current-0.13.tar.bz2'
+  source_sha256 '03bd8cb436830e89407acbc8d4a736356f3780dc863a2e9df593d6c870d0bb55'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.11.7_armv7l/elinks-0.11.7-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.11.7_armv7l/elinks-0.11.7-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.11.7_i686/elinks-0.11.7-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.11.7_x86_64/elinks-0.11.7-chromeos-x86_64.tar.xz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.13_armv7l/elinks-0.13-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.13_armv7l/elinks-0.13-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.13_i686/elinks-0.13-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/elinks/0.13_x86_64/elinks-0.13-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '0d84da6aae40b4ac0df284338634493925ad37ff8a3e5017383de8a8ca8c8db8',
-     armv7l: '0d84da6aae40b4ac0df284338634493925ad37ff8a3e5017383de8a8ca8c8db8',
-       i686: 'cffbb308ddf6d25d74d94d3f17e790b1f0d90ef9e6fea933180c50ef35006152',
-     x86_64: '6473794b1cad1f09965ae540adf4f8fe0e8bb9a957d9d37477ff06e6f99095d0'
+    aarch64: '7478d48810157ace3bf8672f68398ff9ba4fff19f93fde529ebe655ff9c4e2f0',
+     armv7l: '7478d48810157ace3bf8672f68398ff9ba4fff19f93fde529ebe655ff9c4e2f0',
+       i686: '7852bd73a485390c86c0416b28dbfc701dd931f22b0c76fbb478133512b8bd4c',
+     x86_64: '21b68488bdc3dda9286760e087288234ceee0ba147cba8ad8b3b0f67f3205309'
   })
 
-  depends_on 'bz2'
   depends_on 'lzip'
+  depends_on 'openssl' # R
+  depends_on 'zlibpkg' # R
+  depends_on 'bz2' # R
+  depends_on 'expat' # R
 
   def self.build
-    system 'sh autogen.sh'
-    system "./configure --without-x --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
+    system "./configure --without-x #{CREW_OPTIONS}"
     system 'make'
   end
 
   def self.install
-    system "make DESTDIR=#{CREW_DEST_DIR} install"
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    FileUtils.rm_f "#{CREW_DEST_PREFIX}/share/locale/locale.alias"
   end
 end

